src/pages/write-for-egghead/index.mdx

```diff
@@ -0,0 +1,237 @@
+
+<RainbowMDX />
+
+We strongly encourage including visual, code snippets, demos, and interactive elements that [make programming learnable](http://worrydream.com/LearnableProgramming/).
+Wild, interesting experimentation with the medium of the web is very welcome here.
+
+To cite the marvellous [Bret Victor](http://worrydream.com/):
+
+</ProseSection>
+
+
+<blockquote className="relative my-6 text-xl text-neutral-600 pl-6 max-w-2xl font-semibold">
+  "Programming is a way of thinking, not a rote skill" and "people understand
+  what they can see."
+</blockquote>
+
+<ProseSection>
+
+
+Show, don't tell.
+
+### Handy Recipes for Article Types
+
+- Constructive critique and analysis of an existing site or system
+- Common mistakes you see within a specific niche/tool/framework
+- X meets Y
+- How to do X in Y framework
+- A review and summary of existing resources on a topic
+- A creative application of a tool/technique with a code demo
+- Dig in deep on how a system works - "What's behind the curtain?"
+- Interview chat with an expert you find interesting over video and write up notes on the conversation
+- Sketchnotes, diagrams, and illustrated explanations
+- Step-by-step scrollytelling explanations
+
+## Examples of Great articles
+
+- [Full-Bleed Layout Using CSS Grid](https://joshwcomeau.com/css/full-bleed/) by Josh Comeau  
+  Structured around problem → solution, and then digs deeper into edge cases and details. Uses simple clear diagrams, useful interactive components, and lots of examples.
+
+- Another article
+
+- And another
+
+## Payment
+
+We pay a flat rate for each article.  
+The compensation amount is adjusted to the meatiness of the article. Long, in-depth articles that require deep research are compensated more than shorter, snappier articles.
+
+## Cross-posting and Canonical Links
+
+If you already have a blog or digital garden where you publish your writing, you're welcome to cross-post your article there. You just need to make sure the canonical URL links to the egghead version, and at the top of the article link to the original version on egghead.
+
+If you're not sure how to set a canonical URL [read this guide](https://ahrefs.com/blog/canonical-tags/)
+
+## The Process
+
+1. Submit your article idea through [the form](/write-for-egghead#pitch-form)
+2. We'll review your idea and email you if it aligns with our audience and goals
+3. You write up an outline of the main points and sections
+4. We approve the outline
+5. You write a mostly-final article and share it with us in a collaborative writing app like Notion, Dropbox Paper, or Google Docs.
+6. We review and edit your article to help fix any spelling, grammar, style, or technical issues.
+7. We hit publish and promote your article on all the socials.
+8. <span>🤑</span> You get paid <span>🤑</span>
+
+<br />
+
+<section id="pitch-form">
+
+
+## Ready to Pitch?
+
+Fill out this form and we'll get in touch if we're interested.
+
+<form className="max-w-md">
```

Comment:
[suggestion: technical] I'd strongly consider extracting the form as a React component and utilizing Formik for validation.

Reply:
Will do - I'm new to Formik but can copypasta from other examples in the repo to figure it out.

Reply:
I took a good stab at it today but I've hit a wall. There's a bunch of TS errors  I can't decode, and not sure where we're sending form values on submission.

There's a component called `WritingSubmissionForm.tsx` in /write-for-egghead where I moved all the form elements & tried to setup Yup & Formik.

Reply:
Apart from the form issues, the rest of it is good enough to launch. @laurosilvacom Are you fluent in Formik? Unsure who to recruit to help get this over the finish line 😉

Reply:
I haven't use Formik but I'll give it a try :) My initial assumption is that we use Formium to handle submissions. Once on Formium, we can trigger emails, auto-responders, Slack notifications.

Reply:
formik has nothing to do with formium directly @laurosilvacom. it's a React library for making forms

for now we can just leave the form off of this page?